### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by Sevil-h**

### CODE REVIEW

File: app/views/artworks/_navbar.html.erb [hunk 1]
```diff
@@ -14,12 +14,14 @@
           <%= link_to current_user.email, "#", class: "nav-link" %>
         </li>
         <li class="nav-item">
-          <%= link_to "Messages", "#", class: "nav-link" %>
```
Comment (Henniver) [inline app/views/artworks/_navbar.html.erb]: Do you suggest messaging functionality between users?

File: app/views/artworks/_navbar.html.erb [hunk 2]
```diff
@@ -14,12 +14,14 @@
...
+          <%= link_to current_user.username, "#", class: "nav-link" %>
         </li>
         <li class="nav-item dropdown">
           <%= image_tag "https://kitt.lewagon.com/placeholder/users/ssaunier", class: "avatar dropdown-toggle", id: "navbarDropdown", data: { toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
           <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown">
-            <%= link_to "Edit your profil", "#", class: "nav-link" %>
+            <%= link_to "Edit your profil", "#", class: "dropdown-item" %>
```
Comment (Henniver) [inline app/views/artworks/_navbar.html.erb:22]: profilE

**Review by Henniver — CHANGES_REQUESTED** [review]
See comments below. Can you maybe add a screenshot so that we can see what the changes look like?

File: app/views/artworks/_navbar.html.erb (hunk 1 above)
Comment (Sevil-h) [reply app/views/artworks/_navbar.html.erb]: in the beginning I write Messages but then change as a Your Booking and Your Artwork. It looks like 
Screenshot 2021-03-02 at 19 25 08

**Pull request merged**